stdlib/public/runtime/SymbolInfo.cpp

```diff
@@ -76,14 +121,25 @@ llvm::Optional<SymbolInfo> SymbolInfo::lookup(const void *address) {
 #elif defined(_WIN32) && !defined(__CYGWIN__)
   _swift_win32_withDbgHelpLibrary([&] (HANDLE hProcess) {
     if (!hProcess) {
-        return;
+      return;
     }
 
+    // If there are real-world examples of symbol names with Unicode characters
+    // that we need to consider, SymbolInfo can hold a SYMBOL_INFO_PACKAGEW and
+    // call SymFromAddrW(), then lazily convert to UTF-8 on the first call to
+    // getSymbolName() as is done in getFilename().
```

Comment:
Is that really the case?  I thought that the unicodeness was tied to the version of the API used, which should really be Unicode aware.

Reply:
My expectation is that there isn't actually a symbol in the wild with a Unicode character in its name, so I am being lazy here—but if you think there's real-world counter-examples, then I should use `SymFromAddrW()`.

If you're concerned about the conversion from UTF-16 to UTF-8 failing because of some new Unicode character the current OS doesn't know about… well then I wouldn't expect `SymFromAddr()` to succeed there anyway, ya know?

Reply:
Go does use unicode symbols in its name decoration scheme 😭.  But more that the unsuffixed APIs should be assumed to be the `W` variant (we should be building with -DUNICODE=1`).  As such, the storage should be paired with the `W` suffixed type IIRC.  But then again, it may be that there are small nuances that I have forgotten about.

Reply:
No, I think that using the ANSI form for now is fine.  Let's deal with this when we need to.